OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -290,6 +292,19 @@ public interface IProvidesAssetBrowserColorPickerPalettes
 		IEnumerable<string> ColorPickerPaletteNames { get; }
 	}
 
+	public interface IColorPickerManagerInfo : ITraitInfoInterface
+	{
+		(float hMin, float hMax) GetHueRange { get; }
+		(float sMin, float sMax) GetSaturationRange { get; }
+		(float vMin, float vMax) GetValueRange { get; }
```

Comment:
FWIW "Get*" suggests a method, not a property.

Reply:
redacted